app/store/types.ts

```diff
@@ -28,3 +28,9 @@ export type AppThunk<ReturnType = void> = ThunkAction<
   unknown,
   Action<string>
 >;
+
+export enum FeatureFlagOption {
```

Comment:
What do you think about a union type here rather than the enum?

Reply:
The values themselves are used, e.g. as reducer keys, or for flag to name mapping, so I think an enum makes sense? Not sure how you're envisioning a union type would work

Reply:
to be clear, i think what you have here is totally reasonable. I just tend to prefer avoiding enums when possible though, cause numeric enums have some pitfalls, and I find the enum { KEY = 'KEY' } syntax of string enums a bit redundant, so this is purely stylistic I wouldnt take the time to change your commit unless you feel like it.

that said, I would consider something like:

```typescript
type FeatureFlag = 'GoogleImport' | 'CustomUrl' | 'DownloadLocally'

const FeatureConfig: { [key in FeatureFlag]: boolean } = {
  GoogleImport: boolean
  CustomUrl: boolean
  DownloadLocally: boolean
}
```